hash/n64_lodgenet.xml

```diff
@@ -0,0 +1,479 @@
+		<info name="usage" value="Requires 16k EEPROM." />
+		<part name="cart" interface="n64_cart">
+			<dataarea name="rom" size="0x2000000">
+				<rom name="nzsg00.z64" size="0x2000000" crc="e4b47e08" sha1="ecabc86f0de743c02ba3fa0c83ab163538b00be9"/>
+<!--                <rom name="nzsg00.i64" size="0x40" crc="8d23977d" sha1="f882460a888b8db76e8b163fd508d89e7b2e9e00"/> -->
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="zeldaoot">
+		<description>The Legend of Zelda - Ocarina of Time</description>
+		<year>1998</year>
+		<publisher>Nintendo</publisher>
+		<info name="serial" value="czlg00" />
+		<part name="cart" interface="n64_cart">
+			<dataarea name="rom" size="0x2000000">
+				<rom name="czlg00.z64" size="0x2000000" crc="c4d2ac1d" sha1="86ff85565e0e4f78d56f50e73df51ba29d36738a"/>
+<!--                <rom name="czlg00.i64" size="0x40" crc="0eb0198c" sha1="45185a1938db659ee6760117cd4f742aac5e18bf"/> -->
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="marioglf">
+		<description>Mario Golf</description>
+		<year>1999</year>
+		<publisher>Nintendo</publisher>
+		<info name="serial" value="nmfg00"/>
+		<part name="cart" interface="n64_cart">
+			<dataarea name="rom" size="0x2000000">
+				<rom name="nmfg00.z64" size="0x2000000" crc="1c256575" sha1="47c5f700940bb65be1145e7d6543ddf1a816ddda"/>
+<!--                <rom name="nmfg00.i64" size="0x40" crc="df31532f" sha1="51db8e096b345ca306d34d114dc82bf36211db1d"/> -->
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="mariokrt">
+		<description>Mario Kart 64</description>
+		<year>1997</year>
+		<publisher>Nintendo</publisher>
+		<info name="serial" value="nktg00"/>
+		<part name="cart" interface="n64_cart">
+			<dataarea name="rom" size="0xc00000">
+				<rom name="nktg00.z64" size="0xc00000" crc="407edef1" sha1="5b0a0c8b57881f43f9314b427a84d29a46a12607"/>
+<!--                <rom name="nktg00.i64" size="0x40" crc="de24f369" sha1="45ffe8cdac5266f261b7b1ca3d2cca7c48877bea"/> -->
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="mariopr3">
+		<description>Mario Party 3</description>
+		<year>2001</year>
+		<publisher>Nintendo</publisher>
+		<info name="serial" value="nmvg00"/>
+		<info name="usage" value="Requires 16k EEPROM." />
+		<part name="cart" interface="n64_cart">
+			<dataarea name="rom" size="0x2000000">
+				<rom name="nmvg00.z64" size="0x2000000" crc="9f929c0f" sha1="9be79ddb66a6306dec7404e934c6d9fe454f42f4"/>
+<!--                <rom name="nmvg00.i64" size="0x40" crc="80ee8f86" sha1="c2c18a6d2b2c098767d7015f06300a1fc5772a33"/> -->
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="marioten">
+		<description>Mario Tennis</description>
+		<year>2000</year>
+		<publisher>Nintendo</publisher>
+		<info name="serial" value="nm8g00"/>
+		<part name="cart" interface="n64_cart">
+			<dataarea name="rom" size="0x1000000">
+				<rom name="nm8g00.z64" size="0x1000000" crc="02af0a24" sha1="1f76e775c5f2e247d6550aecc2666149c60259
```

Comment:
This line is mangled, need the full sha1 to fix.

Reply:
Fixed now, thank you!